miso-web/src/main/webapp/scripts/list_contact.js

```diff
@@ -80,8 +95,15 @@ ListTarget.contact = (function ($) {
         return {
           name: contactRole.name,
           handler: function () {
-            contact.contactRole = contactRole;
-            Project.addContact(contact);
+            // setting contact variable in ProjectContactDto format
```

Comment:
Done this as we can use the existing add contact function made, and just transfer its information into a new variable with some extra fields (from adding the contact role).